ova/Libraries/elastic_functions.sh

```diff
@@ -0,0 +1,247 @@
+    fi
+
+    systemctl daemon-reload
+    systemctl enable kibana.service
+    systemctl start kibana
+
+    while true; do
+
+        STATUS=$(curl -XGET https://localhost/status -I -s -k | grep HTTP)
+        if [[ "$STATUS" == *"200"* ]]; then
+
+            echo "Kibana is running."
+            break
+        else
+            echo "Waiting for Kibana"
+            sleep 2
+        fi
+    done
+
+}
+
+configure_kibana_app(){
+
+    # Setting up the Kibana plugin API configuration
+    api_config="/tmp/api_config.json"
+    api_time=$(($(date +%s%N)/1000000))
+
+    cp -f ${config_files}/api_config.json  ${api_config}
+
+    sed -i "s/\"manager\": \"wazuh-manager\",/\"manager\": \"$(hostname)\",/" ${api_config}
+    curl -s -XPUT "http://localhost:9200/.wazuh/_doc/${api_time}" -H 'Content-Type: application/json' -d@${api_config}
+    rm -f ${api_config}
+
+    # Configuring default index pattern for Kibana
+    default_index="/tmp/default_index.json"
+    wazuh_major=`echo ${WAZUH_VERSION} | cut -d'.' -f 1`
+
+    cp -f ${config_files}/default_index.json  ${default_index}
+    sed -i "s/{wazuh_major}/${wazuh_major}/g"
+
+    curl -k -POST "https://localhost/api/kibana/settings" -H "Content-Type: application/json" -H "kbn-xsrf: true" -d@${default_index}
+    rm -f ${default_index}
+
+    # Configuring Kibana TimePicker
+    curl -k -POST "https://localhost/api/kibana/settings" -H "Content-Type: application/json" -H "kbn-xsrf: true" -d \
+    '{"changes":{"timepicker:timeDefaults":"{\n  \"from\": \"now-24h\",\n  \"to\": \"now\",\n  \"mode\": \"quick\"}"}}'
+
+    # Do not ask user to help providing usage statistics to Elastic
+    curl -k -POST "https://localhost/api/telemetry/v1/optIn" -H "Content-Type: application/json" -H "kbn-xsrf: true" -d '{"enabled":false}'
+
+}
+
+enable_geo_ip_7(){
+
+    # Enable GeoIP
+    geoip="/tmp/geoip.json"
+    cp -f ${config_files}/geoip.json  ${geoip}
+
+    curl -X PUT "localhost:9200/_ingest/pipeline/geoip" -H 'Content-Type: application/json' -d@${geoip}
+
+    systemctl restart filebeat
+}
+
+install_jdk_6(){
+
+    yum install -y java-1.8.0-openjdk
+}
+
+insert_elasticsearch_template_6(){
+
+  until $(curl "localhost:9200/?pretty" --max-time 2 --silent --output /dev/null); do
+    echo "Waiting for Elasticsearch..."
+    sleep 2
+  done
+
+  # Insert the template
+  curl https://raw.githubusercontent.com/wazuh/wazuh/v${WAZUH_VERSION}/extensions/elasticsearch/wazuh-elastic6-template-alerts.json -so template.json
+  sed -i 's#"index.refresh_interval": "5s"#"index.refresh_interval": "5s",\n    "number_of_shards": 1,\n    "number_of_replicas":0#g' template.json
+  curl -s -XPUT "http://localhost:9200/_template/wazuh" -H 'Content-Type: application/json' -d @template.json
+}
+
+install_logstash_6(){
+
+    yum install logstash-${2} -y
```

Comment:
This line should be `yum install logstash-${ELK_VERSION}`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/06b971366820c9d9cbf359ba4c3b01c427772310